Microsoft.Toolkit.Uwp.Connectivity/BluetoothLEHelper/ObservableBluetoothLEDevice.cs

```diff
@@ -357,7 +357,7 @@ private set
         /// Gets the bluetooth address of this device as a string
         /// </summary>
         /// <value>The bluetooth address as string.</value>
-        public string BluetoothAddressAsString => DeviceInfo.Properties["System.Devices.Aep.DeviceAddress"].ToString();
+        public string BluetoothAddressAsString => DeviceInfo.Properties["System.Devices.Aep.DeviceAddress"]?.ToString() ?? string.Empty;
```

Comment:
It makes much more semantic sense to return null here, not an empty string.

Reply:
```suggestion
        public string BluetoothAddressAsString => DeviceInfo.Properties["System.Devices.Aep.DeviceAddress"]?.ToString();
```

Reply:
@GeertvanHorrik are you good with this change?  If so, you can just click the `Commit suggestion` button and then we'll merge it in.  Thanks!

Reply:
I would opt-in for null (since the address is null as well, meaning no value (where string.Empty *is* a value)), but I am fine with empty string as well.

Reply:
@azchohfi want to sign-off and merge?